website/content/en/docs/building-operators/golang/operator-scope.md

```diff
@@ -89,42 +97,53 @@ in `role_binding.yaml` are used to grant the operator permissions to access and
 For the purposes of this doc, the other RBAC manifests `<kind>_editor_role.yaml`, `<kind>_viewer_role.yaml`, 
 and `auth_proxy_*.yaml` are not relevant to changing the operator's resource permissions.
 
-### Changing the permissions 
+### Changing the permissions to Namespaced
+
+- Inform the Namespace to the [Manager][ctrl-manager] 
+
+By default, the [Manager][ctrl-manager] in the `main.go` is initialized with no Namespace option specified which mean that will watch all Namespaces. Update the `NewManager` to inform the Namespace, in our `Memcahced` example it would like: 
+
+```go
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+    Namespace:          "memcached-operator-system", // operator namespace
+})
+...
+```
+
+- Change the scope of the RBAC permissions from cluster-wide to a specific namespace
 
-To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use `Role`s 
+You will need to use `Role`s 
 and `RoleBinding`s instead of `ClusterRole`s and `ClusterRoleBinding`s, respectively.
  
 [`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/memcached_controller.go`)
 are used to generate the operator's [RBAC ClusterRole][rbac-clusterrole] (e.g `config/rbac/role.yaml`). The default
  markers don't specify a `namespace` property and will result in a `ClusterRole`.
 
-Update the RBAC markers to specify a `namespace` property so that `config/rbac/role.yaml` is generated as a `Role`
- instead of a `ClusterRole`.
-
-Replace:
+Update the [`RBAC markers`][rbac-markers] in `<kind>_controller.go` with `namespace=<namespace>` where the `Role` should be applied such as:
```

Comment:
Nit:
...with `namespace=<namespace>` where the `Role` is to be applied, such as: